app/src/components/main/attributeList/attributeList.jsx

```diff
@@ -0,0 +1,62 @@
+import { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { FormattedMessage } from 'react-intl';
+import { EditableAttribute } from './editableAttribute';
+import styles from './attributeList.scss';
+
+const cx = classNames.bind(styles);
+
+const createEditHandler = (attributes, index, onChange) => (newAttribute) => {
+  const newAttributes = [...attributes];
+  newAttributes[index] = newAttribute;
+  onChange(newAttributes);
+};
+
+const createRemoveHandler = (attributes, index, onRemove) => () => {
+  const newAttributes = [...attributes];
+  newAttributes.splice(index, 1);
+  onRemove(newAttributes);
+};
+
+const isEditMode = (currentAttribute, editedAttribute) =>
+  editedAttribute &&
+  currentAttribute.key === editedAttribute.key &&
+  currentAttribute.value === editedAttribute.value;
+
+export const AttributeList = ({ attributes, editedAttribute, onChange, onEdit, onAddNew }) => (
+  <Fragment>
+    {attributes
+      .filter((attr) => !attr.system)
```

Comment:
You can use `notSystemAttributePredicate` from `common/utils/attributeUtils` like in the `pages/inside/common/itemInfo/attributesBlock`

Reply:
fixed